### PR TITLE
update citations and biblio section

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

#549 

### Please describe the goal of this pull request and the changes that were made.

update section for 11ty changes

### Additional Comments

I added the underline HTML to give people a sense of what part will actually be linked, which I think is important. Is the display text section correct?

### CODE REVIEW

File: content/docs-v1/citation-bibliographies.md [hunk 1]
```diff
@@ -33,72 +33,110 @@ entries:
...
+```yaml
+  bibliography:
+    displayOnPage: false
+```
 
-Your publication will automatically include a page-level bibliography listing all works that were cited on that page using the `q-cite` shortcode. A heading can be customized to go above the page bibliography with the `biblioHeading` parameter in your `config.yaml` file. The default is "Bibliography".
+If you do not want to use in-text citations but still want to include a page-level bibiliography you have a two options:
 
-Additionally, to create a complete bibliography for your entire publication, from all the entries in the project's `references.yaml` file, you can create a `bibliography.md` file and use the `q-bibliography` shortcode. The resulting bibliography will be output as a page in your publication. By default, your bibliography will automatically be ordered according to the `full` value in your `references.yaml` file.
+- Add a page-level biblio using Markdown
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:116]: Though I suggested this as an option in that GitHub issue, I don't think we should necessarily include it here. Better to assume users will work with refereences.yaml and not float the idea of just dumping it into markdown. Or, if you do want to keep it, I'd make it the second option, but then elaborate just a little more ....

> Skip the use of references.yaml, and instead add your bibliography directly into the page with Markdown, using list or definition list formatting

Markdown lists we have documented but we might not have definition lists included? But they work, here's a sample: 

```
apple
: a fruit you give teachers

banana
: mushy to eat, funny to slip on

orange
: a fruit and a color
```

File: content/docs-v1/citation-bibliographies.md [hunk 2]
```diff
@@ -33,72 +33,110 @@ entries:
...
+  - "Lynd 1929"
 ```
 
-{{< q-figure id="modern-bibliography" >}}
+For the latter option, the order they’re listed in doesn’t matter. Quire will sort them all alphabetically unless a `sort_as` attribute has been added in the `references.yaml` file.
 
+### Complete Bibliography
 
-### Display the Short Reference in Bibliographies
+Additionally, to create a complete bibliography for your entire publication, from all the entries in the project's `references.yaml` file, you would add a `bibliography.md` file to your `content` directory. Then insert the `{% bibliography %}` shortcode to this `.md` file. The resulting bibliography will be output as a page in your publication. By default, your bibliography will automatically be ordered according to the `full` value in your `references.yaml` file unless the `sort_as` attribute has been applied.
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:133]: See note above about using `layout: bibliography` instead

File: content/docs-v1/citation-bibliographies.md [hunk 3]
```diff
@@ -33,72 +33,110 @@ entries:
...
+
+{{< q-figure id="modern-bibliography" >}}
 
-Bibliographies displayed automatically at the bottom of pages, and those generated with the `q-bibliography` shortcode, can be a list of the full version of the reference, or can include the short version as well. This is controlled globally (all bibliographies in the project have to be the same format) in the `config.yaml` file with the `displayBiblioShort` property, can be set to `"true"` or `"false"`.
+A heading can be customized to go above the page bibliography in your `content/_data/config.yaml` file. The default heading is "Bibliography".
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:137]: This part is still important to keep but applies to page-level bibliographies specifically

**Review by geealbers — CHANGES_REQUESTED** [review]

File: content/docs-v1/citation-bibliographies.md (hunk 2 above)
Comment (geealbers) [reply content/docs-v1/citation-bibliographies.md:133]: It wouldn't. `lauyout: bibliography` will create a page that has the full bibliography for the whole book (all the references in references.yaml). I don't think there's a use case for having that full biblio appear on an essay or entry page. Those will have their own page-level biblios based on usage of `% cite %`.

File: content/docs-v1/citation-bibliographies.md [hunk 4]
```diff
@@ -5,13 +5,13 @@ type: essay
 abstract: "Cite sources with pop-ups and generate reference lists"
 ---
 
-In-text citations and bibliographies are all available in Quire. Designed to meet scholarly needs and multiple citation styles, they are easy to implement in your publications. While bibliographic references are formatted in YAML and stored in a YAML file (you can consult our [YAML syntax fundamentals](/docs-v1/fundamentals/) for more information), citation and bibliography {{< q-def "shortcodes" >}} are used to integrate the references in your publication.
+In-text citations and bibliographies are all available in Quire. Designed to meet scholarly needs and multiple citation styles, they are easy to implement in your publications. Bibliographic references are formatted in YAML and stored in a YAML file (you can consult our [YAML syntax fundamentals](/docs-v1/fundamentals/) for more information) while citation and bibliography {{< q-def "shortcodes" >}} are used to integrate these references in your publication.
```
[the remark this answers is not in the document]
Comment (Erin-Cecele) [reply content/docs-v1/citation-bibliographies.md:8]: Fixing this. I noticed in CC that there is also a `class: backmatter` is this required for a bibliography.md? Or is it just a Getty pubs styling preference?

[the remark this answers is not in the document]
Comment (geealbers) [reply content/docs-v1/citation-bibliographies.md:8]: I have some open questions about Quire's use of the `backmatter` class. I don't think it's stable/tested enough to document yet. So I'd just ignore it for now

File: content/docs-v1/citation-bibliographies.md [hunk 5]
```diff
@@ -33,72 +33,110 @@ entries:
...
-```go
-{{</* q-cite "Faure 1909" "" "1909" */>}}
+```md
+{% cite 'Faure 1909' '' '1909' %}
 ```
 
-The text element between the Author Date citation and the page can be changed with the `citationPageLocationDivider` property in `config.yaml`. The humanities tend to favor comma separation (which is the default in Quire), whereas the sciences typically favor a colon.
+{{< q-class "box tip" >}}
+- The text element between the in-text citation and the page can be changed in the `content/_data/config.yaml` file. The humanities tend to favor comma separation (which is the default in Quire), whereas the sciences typically favor a colon.
+```yaml
```
[the remark this answers is not in the document]
Comment (Erin-Cecele) [reply content/docs-v1/citation-bibliographies.md:84]: FYI I couldn't get this to display properly so I removed the tip

**Comment by Erin-Cecele** [comment]
@geealbers changes have been made and are ready for review.

File: content/docs-v1/citation-bibliographies.md [hunk 6]
```diff
@@ -5,13 +5,11 @@ type: essay
 abstract: "Cite sources with pop-ups and generate reference lists"
 ---
 
-In-text citations and bibliographies are all available in Quire. Designed to meet scholarly needs and multiple citation styles, they are easy to implement in your publications. While bibliographic references are formatted in YAML and stored in a YAML file (you can consult our [YAML syntax fundamentals](/docs-v1/fundamentals/) for more information), citation and bibliography {{< q-def "shortcodes" >}} are used to integrate the references in your publication.
+In-text citations and bibliographies are available in Quire. They are easy to implement and designed to meet scholarly needs and multiple citation styles. Bibliographic references are stored in a YAML file while citation and bibliography {{< q-def "shortcodes" >}} are used to integrate these references into your publication.
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:8]: This still references a bibliography shortcode

File: content/docs-v1/citation-bibliographies.md [hunk 7]
```diff
@@ -33,72 +31,102 @@ entries:
...
-
-```yaml
-entries:
-  - id: "Mütter 1851"
-    full: "Mütter, Edmond. *Journal des Goncourt: Mémoires de la vie littéraire.* Paris; G. Charpentier et cie, 1851."
-    sort: "Mutter"
-```
-
-These references can then be called individually from within text using the `q-cite` shortcode, or in their entirety as a generated bibliography using the `q-bibliography` shortcode. Both of which are detailed below.
+These references can then be called individually from within text using the `{% cite %}` shortcode or they can appear in their entirety as a page of your publication by adding a `bibliography.md` file to your project. Both options are detailed below.
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:38]: > by adding a `bibliography.md` file to your project

The file doesn't need to be named `bibliography.md` it just needs to have `layout: bibliography`

File: content/docs-v1/citation-bibliographies.md [hunk 8]
```diff
@@ -33,72 +31,102 @@ entries:
...
+If you want to suppress the page level bibliography but keep the in-text citations, go into the `content/_data/config.yaml` and change the value to `false`.
+
+```yaml
+  bibliography:
+    displayOnPage: false
+```
+
+### Complete Bibliography
+
+You can also create a complete bibliography for your entire publication from all the entries in the project's `references.yaml` file. To do this add a `bibliography.md` file to your `content` directory and add the `layout` type of `bibliography` to your page YAML.
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:122]: Same note about `bibliography.md` filename as above

File: content/docs-v1/citation-bibliographies.md [hunk 9]
```diff
@@ -33,72 +31,102 @@ entries:
...
+A heading can be customized to go above this bibliography in your `content/_data/config.yaml` file. The default heading is "Bibliography".
 
-Bibliographies displayed automatically at the bottom of pages, and those generated with the `q-bibliography` shortcode, can be a list of the full version of the reference, or can include the short version as well. This is controlled globally (all bibliographies in the project have to be the same format) in the `config.yaml` file with the `displayBiblioShort` property, can be set to `"true"` or `"false"`.
+```yaml
+bibliography:
+  heading: Bibliography
+```
+
+{{< q-class "box tip" >}}
+- If entries as missing from your page-level bibliography or you see output in your command-line shell referencing missing entries, then there is most likely an incorrect or absent `{% cite %}` shortcode. The `id` used in the `{% cite %}` shortcode must exactly match the `id` in the `references.yaml` file for it to work.
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:110]: > If entries as missing ...

Should presumably be "If entries are missing ..."

File: content/docs-v1/citation-bibliographies.md [hunk 10]
```diff
@@ -33,72 +31,102 @@ entries:
...
+{{< /q-class >}}
+
+If you want to suppress the page level bibliography but keep the in-text citations, go into the `content/_data/config.yaml` and change the value to `false`.
```
Comment (geealbers) [inline content/docs-v1/citation-bibliographies.md:113]: > page level
 
"page-level" hyphenated

**Review by geealbers — CHANGES_REQUESTED** [review]

**Comment by Erin-Cecele** [comment]
Ok, issues resolved. Sorry about that.

**Review by geealbers — APPROVED** [review]

**Pull request merged**